src/main/java/com/dev/cinema/controllers/MovieController.java

```diff
@@ -0,0 +1,37 @@
+package com.dev.cinema.controllers;
+
+import com.dev.cinema.mappers.MovieMapper;
+import com.dev.cinema.model.dto.MovieRequestDto;
+import com.dev.cinema.model.dto.MovieResponseDto;
+import com.dev.cinema.service.MovieService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movies")
+public class MovieController {
+    public final MovieService movieService;
+    public final MovieMapper movieMapper;
```

Comment:
these fields should be private in all controllers and in mappers

Reply:
done